### PR TITLE
Converted UK Ireland accent recognition to work with Keras Core

### PR DESCRIPTION
**Pull request by fbadine**
git diff with the original example gave the following

@@ -1,8 +1,9 @@
 """
 Title: English speaker accent recognition using Transfer Learning
 Author: [Fadi Badine](https://twitter.com/fadibadine)
+Converted to Keras Core by: [Fadi Badine](https://twitter.com/fadibadine)
 Date created: 2022/04/16
-Last modified: 2022/04/16
+Last modified: 2023/07/19
 Description: Training a model to classify UK & Ireland accents using feature extraction from Yamnet.
 Accelerator: GPU
 """
@@ -40,7 +41,7 @@ pip install -U -q tensorflow_io
 """
 
 SEED = 1337
-EPOCHS = 100
+EPOCHS = 1
 BATCH_SIZE = 64
 VALIDATION_RATIO = 0.1
 MODEL_NAME = "uk_irish_accent_recognition"
@@ -104,7 +105,7 @@ import pandas as pd
 import tensorflow as tf
 import tensorflow_hub as hub
 import tensorflow_io as tfio
-from tensorflow import keras
+import keras_core as keras
 import matplotlib.pyplot as plt
 import seaborn as sns
 from scipy import stats
@@ -333,7 +334,7 @@ keras.backend.clear_session()
 
 
 def build_and_compile_model():
-    inputs = keras.layers.Input(shape=(1024), name="embedding")
+    inputs = keras.layers.Input(shape=(1024,), name="embedding")
 
     x = keras.layers.Dense(256, activation="relu", name="dense_1")(inputs)
     x = keras.layers.Dropout(0.15, name="dropout_1")(x)
@@ -408,7 +409,7 @@ early_stopping_cb = keras.callbacks.EarlyStopping(
 )
 
 model_checkpoint_cb = keras.callbacks.ModelCheckpoint(
-    MODEL_NAME + ".h5", monitor="val_auc", save_best_only=True
+    MODEL_NAME + ".keras", monitor="val_auc", save_best_only=True
 )
 
 tensorboard_cb = keras.callbacks.TensorBoard(

### CODE REVIEW

File: examples/keras_io/tensorflow/audio/speaker_recognition_using_cnn.py [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+- The noise samples in the dataset need to be resampled to a sampling rate of 16000 Hz
+before using the code in this example. In order to do this, you will need to have
+installed `ffmpg`.
+"""
+
+"""
+## Setup
+"""
+
+import os
```
Comment (fchollet) [inline examples/keras_io/tensorflow/audio/speaker_recognition_using_cnn.py:42]: Set the KERAS_BACKEND env variable since the example is TF specific (same for the other file)

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: examples/keras_io/tensorflow/audio/speaker_recognition_using_cnn.py (hunk 1 above)
Comment (fbadine) [reply examples/keras_io/tensorflow/audio/speaker_recognition_using_cnn.py:42]: KERAS_BACKEND added to both files

**Review by fchollet — APPROVED** [review]
LGTM, thanks!
